_locales/en/messages.json

```diff
@@ -1775,6 +1775,9 @@
 			},
 			"span_end": {
 				"content": "</span>"
+			},
+			"dollar_sign": {
+				"content": "$1"
```

Comment:
@zarembsky since the dollar sign is static content, can we just pass `"content": "$"` here? This way we don't have to remember to pass it in when we use this string.